### ISSUE TITLE
Possible to use fread for load_dam?

### ISSUE THREAD

**Issue by brelwill** [rethomics/damr#14]
I'm just wondering if it's possible to modify load_dam to use the fread function? With large raw data files like the ones were using, it seems to only use one core when processing the data. Not sure if this is compatible with what you've written, but I thought it was worth an ask.

**Comment by qgeissmann** [comment]
It is definitely possible and probably good. Indeed, we avoid dependency on `readr`.
At first, I implemented as much as possible using the resources from the tidyverse, mainly to learn. In the end, I reimplemented most of it in `data.table` for consistency. It would definitely be good to make a last push for it.  (https://github.com/rethomics/damr/blob/master/R/read-dam-file.R, line 47).
Are you confident the bottleneck is here (see profiling http://adv-r.had.co.nz/Profiling.html#measure-perf)?
